pkg/scheduler/scheduler.go

```diff
@@ -267,19 +313,10 @@ func (s *Scheduler) getScheduleType(key string) ScheduleType {
 			return FirstSchedule
 		}
 
-		policyNamespace := util.GetLabelValue(resourceBinding.Labels, util.PropagationPolicyNamespaceLabel)
-		policyName := util.GetLabelValue(resourceBinding.Labels, util.PropagationPolicyNameLabel)
-
-		policy, err := s.policyLister.PropagationPolicies(policyNamespace).Get(policyName)
-		if err != nil {
-			return Unknown
-		}
-		placement, err := json.Marshal(policy.Spec.Placement)
+		_, policyPlacementStr, err := s.getPlacement(resourceBinding)
```

Comment:
Yeah, much better now. 
We can use this function for `ClusterResourceBinding` below, right? why not?

Reply:
Yes, I consider it when implementing, no specific reason to not adopt it.
So you prefer to make them consistent?

Reply:
I consider it when implementing, since for `ClusterPropagationPolicy` case, it's `ClusterResourceBinding` rather than `ResourceBinding`, so it's will complex the util function. and the logic for `ClusterResourceBinding` is simple and straightforward, so I leave it there.